ocaml/xapi/vm_platform.ml

```diff
@@ -100,6 +100,14 @@ let is_true ~key ~platformdata ~default =
   with Not_found ->
     default
 
+let is_valid_device_model ~key ~platformdata =
+  try
+    match List.assoc key platformdata with
+    | "qemu-upstream-compat" -> true
+    | v -> false
```

Comment:
I'd use the wildcard `_` as `v` is not used.

Reply:
Ok

Reply:
updated.